java/src/jmri/jmrix/loconet/AbstractAlmImplementation.java

```diff
@@ -205,24 +205,28 @@ void writeMsg(LocoNetMessage msg) {
 
     /**
      * Notify possible subclass that a block has changed.
+     * @param block - something about a block
      */
     public void noteChanged(int block) {
     }
 
     /**
      * Notify possible subclass that a read cmd is being handled
+     * @param block - something about a block
      */
     public void noteReadCmd(int block) {
     }
 
     /**
      * Notify possible subclass that a read reply is being handled
+     * @param block - something about a block
```

Comment:
Not sure "something about a block" is a comment worth having.  Are you putting these in because you were getting a warning of some kind?

Reply:
Yes, I was trying to remove a complaint that the @param wasn't included.  I will plan to back-out that useless change (and any others I can find), and live with the related javadoc warning.

Reply:
We have *always* allowed the major IDEs (Eclipse and NetBeans) to warn on missing parameters, but we just silence those in CI because otherwise there are just way too many warnings (and we only error the CI if there are more than 2300 warnings).

One note is that starting with Java 8, javadoc became more stringent about what triggers a warning or an error. It's good to have the warnings in place, because undocumented parameters aren't helpful, although at the same time, that does not mean that documented parameters are helpful.

Reply:
BTW, running `ant javadoc-lint` (which does warn on missing Javadoc tags), there are now only 1656 missing tags in the Javadocs, vs the nearly 6000 two years ago, so I'd be inclined to leave those warnings in place.

I'd change the parameter description, since the current one doesn't mean anything to me, to something like "LocoNet block address" (and leave the dash off - Javadoc puts one in place automatically and then renders the dash on the param tag as well).